BusinessTampereTrafficMonitoring/traffic_lights/api_client.py

```diff
@@ -29,6 +28,15 @@ class TrafficLightCycle(Base):
 
 class TrafficLightAPIClient:
     def __init__(self, url: str, monitored_devices: List[str], db: str):
+        """
+        Constructor for initializing TrafficLightAPIClient class
+
+        #Parameter:
+        # argument1: url (String)
```

Comment:
The docstring should say what URL this is. It is the API end point for the traffic lights API `"http://trafficlights.tampere.fi/api/v1/deviceState/"`